apps/ledger-live-mobile/src/components/TabBar/Transfer.tsx

```diff
@@ -146,10 +146,13 @@ export function TransferTabIcon() {
   }, [openAnimValue, track, readOnlyModeEnabled]);
 
   const closeModal = useCallback(() => {
-    setIsOpened(false);
+    const animCallback = () => {
+      setIsOpened(false);
+    };
     openAnimValue.value = withTiming(2, animParams, finished => {
       if (finished) {
         openAnimValue.value = 0;
+        runOnJS(animCallback)();
```

Comment:
![michael-scott-the-office](https://user-images.githubusercontent.com/91890529/220575296-8c0a14fa-2855-455a-8cc7-8a9e1765139f.gif)

Reply:
I miss the office